### DISCUSSION TITLE
Loop elements

### DISCUSSION THREAD

**Discussion by CharlesdeLalaing** [emailjs-com/help#55] [Q&A]
Hey team, can i loop elements? I a multiple choice select on my website, i would to render them in the email.js template. For now i get this error:Template: One or more dynamic variables are corrupted. Here is my code: Vous avez reçu un nouveau message de {{fullName}} ({{company}})
Cet e-mail provient du formulaire "Produits" :

Nom complet : {{fullName}}

E-mail : {{email}}

Entreprise : {{company}}

Téléphone : {{phone}}

Sujet : {{sujet}}

Produits :

{{#if productChoice}} {{#each productChoice}}
{{this}}
{{/each}} {{else}}
Aucun produit sélectionné
{{/if}}
Message : {{message}}

Cordialement,

I you could help me that would be great :)
Kind regards,
Charles

**Comment by xr0master** [comment]
Our template syntax is more like the [mustache template](https://github.com/janl/mustache.js).

View:
```
{
  "musketeers": ["Athos", "Aramis", "Porthos", "D'Artagnan"]
}
```
Template:
```
{{#musketeers}}
* {{.}}
{{/musketeers}}
```
Output:
```
* Athos
* Aramis
* Porthos
* D'Artagnan
```

**Comment by xr0master** [comment]
And if you need conditions, the code will be like below.
View:
```
{
  "person": false
}
```

Template:
```
Shown.
{{#person}}
Never shown!
{{/person}}
```

Output:
```
Shown.
```

**Comment by xr0master** [comment]
So in your case, the template will be

```
{{#productChoice}}
{{*}}
{{/productChoice}}
{{^productChoice}}
Aucun produit sélectionné
{{/productChoice}}
```

**Comment by xr0master** [comment]
We have it in our docs.
Closed.
